draft-ietf-quic-multipath.md

```diff
@@ -95,12 +95,9 @@ connection.
 This proposal is based on several basic design points:
 
   * Re-use as much as possible mechanisms of QUIC version 1. In
-particular, this proposal uses path validation as specified for QUIC
+particular, this proposal uses packet headers and path validation as specified for QUIC
 version 1 and aims to re-use as much as possible of QUIC's connection
 migration.
-  * Use the same packet header formats as QUIC version 1 to avoid the
-risk of packets being dropped by middleboxes (which may only support
-QUIC version 1)
```

Comment:
I agree with Mirja that the reuse of packet headers and path validation is to distinguish between different needs, and I fully understand the purpose of this PR. So how about:

_* Use the same packet header formats as QUIC version 1 to keep the same connectivity with QUIC version 1 through middleboxes._